content/messaging/agent-chat-api/changelog/index.mdx

```diff
@@ -47,6 +47,7 @@ This document is the record of all the changes in the **Agent Chat API**, Web an
 - `scopes` in **Multicast** has been renamed to `recipients`.
 - There're new fields `previous_thread_id` and `next_thread_id` in the [**Thread**](/messaging/agent-chat-api/v3.2/#threads) data structure.
 - Fields `order` and `timestamp` in the [**Thread**](/messaging/agent-chat-api/v3.2/#threads) data structure were replaced with the new field, `created_at` (date & time in microseconds in UTC).
+- **Get Chat Threads** evolved into **Get Chat** ([Web](/messaging/agent-chat-api/v3.2/#get-chat) & [RTM](/messaging/agent-chat-api/v3.2/rtm-reference/#get-chat)) and allows now to get chat with specific thread or the last one. It also changed it shape: `order` field was removed as well as nesting was simplified (no `chat` key).
```

Comment:
I rephrased it a little. Personally, I would skip this part:

> (by default, it returns the most recent thread if not specified otherwise)

and included it only in the method description.